tests/Integration/Command/EnqueueWebhookCommandTest.php

```diff
@@ -0,0 +1,102 @@
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        HttpClientStub::$responseStatusCode = 201;
+        ActiveCampaignWebhookClientStub::$activeCampaignResources = [
+            [
+                'url' => 'https://other.com/webhook/activecampaign/list-status',
+                'listid' => '43',
+                'webhook' => new WebhookResponse(45),
+            ],
+        ];
+        $this->channelRepository = self::getContainer()->get('sylius.repository.channel');
+
+        $fixtureLoader = self::getContainer()->get('fidry_alice_data_fixtures.loader.doctrine');
+        $fixtureLoader->load([
+            self::FIXTURE_BASE_DIR . '/channels.yaml',
+        ], [], [], PurgeMode::createDeleteMode());
+    }
+
+    public function test_that_it_enqueues_webhook(): void
+    {
+        $channel = $this->channelRepository->findOneBy(['code' => 'fashion_shop']);
+        self::assertNotNull($channel->getId());
+        $commandTester = $this->executeCommand([
+            'channel-id' => $channel->getId(),
+        ], []);
+        self::assertEquals(0, $commandTester->getStatusCode());
+
+        $transport = self::getContainer()->get('messenger.transport.main');
+        /** @var Envelope[] $messages */
+        $messages = $transport->get();
+        $this->assertCount(1, $messages);
+        $message = $messages[0];
+        $this->assertInstanceOf(WebhookCreate::class, $message->getMessage());
+        $this->assertEquals($channel->getId(), $message->getMessage()->getChannelId());
+    }
+
+    public function test_that_it_enqueues_webhook_interactively(): void
+    {
+        $channel = $this->channelRepository->findOneBy(['code' => 'fashion_shop']);
+        self::assertNotNull($channel->getId());
+        $commandTester = $this->executeCommand([], [
+            $channel->getId(),
+        ]);
+        self::assertEquals(0, $commandTester->getStatusCode());
+
+        $transport = self::getContainer()->get('messenger.transport.main');
+        /** @var Envelope[] $messages */
+        $messages = $transport->get();
+        $this->assertCount(1, $messages);
+        $message = $messages[0];
+        $this->assertInstanceOf(WebhookCreate::class, $message->getMessage());
+        $this->assertEquals($channel->getId(), $message->getMessage()->getChannelId());
+    }
+
+    public function test_that_it_enqueues_all_webhooks(): void
+    {
+        $commandTester = $this->executeCommand([
+            '--all' => true,
+        ]);
+        self::assertEquals(0, $commandTester->getStatusCode());
+
+        $fashionChannel = $this->channelRepository->findOneBy(['code' => 'fashion_shop']);
+        $digitalChannel = $this->channelRepository->findOneBy(['code' => 'digital_shop']);
+        $transport = self::getContainer()->get('messenger.transport.main');
+        /** @var Envelope[] $messages */
+        $messages = $transport->get();
+        $this->assertCount(2, $messages);
```

Comment:
Why it didn't enqueue "other_shop" channel?

Reply:
All the channels have the same list id associated, but the other_shop channel won't be enqueued 'cause there is already a [webhook for that route and that list](https://github.com/webgriffe/SyliusActiveCampaignPlugin/pull/52/files#diff-bffa57b3c1d29ca765e7fc7d33f4839a92a1d68bad9624f3111b42a31085bedeR25).

Reply:
Ohh ok, missed it. Good